### PR TITLE
Api 16220/create endpoint for loast full report

### PR DESCRIPTION
**Pull request by stevelong00**
Api 16220/create endpoint for loast full report

- new endpoint created
- added unit tests
- added integration tests

### CODE REVIEW

**Review by mheusgit — COMMENTED** [review]
Looks good to me.  I'll hold off on approving since I wrote a chunk of this

**Review by christineyost — COMMENTED** [review]
I have a couple suggestions, but if this needs to go in without those I'd be ok with that

**Review by dsteeve — DISMISSED** [review]
looks great! 👍

File: src/loast/loast.filter.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+): Record<string, ExampleGroupResult> {
+  const result = {};
+
+  for (let x = 0; x < exgroups.length; x++) {
+    const exgroup = exgroups[x];
+    result[exgroup.name] = {
+      errors: exgroup.errors,
+      warnings: exgroup.warnings,
+    };
+  }
```
[the remark this answers is not in the document]
Comment (stevelong00) [reply src/loast/loast.filter.ts:22]: `forEach` doesn't seem to play nice with the mikro-orm `Collection`.  Instead I used `for...of` as recommended in the [mikro-orm documentation.](https://mikro-orm.io/docs/collections).  It's a little cleaner too!

File: src/loast/loast.controller.ts [hunk 2]
```diff
@@ -15,8 +15,8 @@ import LoastServiceTransactional from './loast.service.transactional';
 import { LoastResultQuery } from './interfaces/loast.common.interface';
 import { loastResultRequestBodySchema } from './loast.schema';
 import { LoastRequestBody } from './interfaces/loast.input.interface';
+import FilterLoastResultResponse from './loast.filter';
```
[the remark this answers is not in the document]
Comment (stevelong00) [reply src/loast/loast.controller.ts]: Fixed

File: src/loast/loast.filter.ts [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+function filterEndpoints(
+  endpoints: Collection<LoastResultEndpointModel, unknown>,
+  includeFailures: boolean,
+): Record<string, LoastOperationResult> {
+  const result = {};
+
+  for (let x = 0; x < endpoints.length; x++) {
+    const endpoint = endpoints[x];
```
[the remark this answers is not in the document]
Comment (stevelong00) [reply src/loast/loast.filter.ts]: See above

**Review by christineyost — APPROVED** [review]
Looks good to me!

**Review by dsteeve — APPROVED** [review]
looks good!

**Pull request merged**